analyzers/tests/SonarAnalyzer.Test/TestCases/MethodsShouldNotHaveIdenticalImplementations.CSharp9.cs

```diff
@@ -95,13 +95,13 @@ public static class TypeConstraints
 
     public static int Use<T>(T? value) where T : class => 2;
 
-    public static void First<T>(T? value) where T : struct // Secondary
+    public static void First<T>(T? value) where T : struct
     {
         var x = Use(value);
         Console.WriteLine(x);
     }
 
-    public static void Second<T>(T? value) where T : class  // Noncompliant - FP, method looks the same but different overloads are called due to the type constraints used. See: https://github.com/SonarSource/sonar-dotnet/issues/7068
+    public static void Second<T>(T? value) where T : class  // Compliant, method looks the same but different overloads are called due to the type constraints used. See: https://github.com/SonarSource/sonar-dotnet/issues/7068
```

Comment:
You can remove the link to the issue now that it's fixed.
```suggestion
    public static void Second<T>(T? value) where T : class  // Compliant, method looks the same but different overloads are called due to the type constraints used.
```